sw/host/opentitanlib/src/app/config/hyperdebug_chipwhisperer.json

```diff
@@ -237,5 +247,22 @@
       "name": "console",
       "alias_of": "UART2"
     }
+  ],
+  "strappings": [
+    {
+      "name": "PRERESET_DFT_DISABLE",
+      "pins": [
+        {
+          "name": "DFT_STRAP0",
+          "mode": "PushPull",
+          "level": false
+        },
+        {
+          "name": "DFT_STRAP1",
+          "mode": "PushPull",
+          "level": false
+        }
+      ]
+    }
```

Comment:
If we use the above, then we'd have straps for enabling "CONSOLE_TX" in tests and whatever DFT config:
```suggestion
    {
      "name": "CONSOLE_TX",
      "pins": [
        {
          "name": "DFT_STRAP0",
          "mode": "Alternate",
          "pull_mode": "None"
        }
      ]
    },
    {
      "name": "DFT_STRAP_CFG1",
      "pins": [
        {
          "name": "DFT_STRAP0",
          "mode": "Input",
          "pull_mode": "PullUp"
        },
        {
          "name": "DFT_STRAP1",
          "mode": "Alternate",
          "pull_mode": "PullDown",
        }
      ]
    },
    {
      "name": "DFT_STRAP_CFG2",
      "pins": [
        {
          "name": "DFT_STRAP0",
          "mode": "Input",
          "pull_mode": "PullDown"
        },
        {
          "name": "DFT_STRAP1",
          "mode": "Alternate",
          "pull_mode": "PullUp",
        }
      ]
    },
    {
      "name": "DFT_STRAP_CFG3",
      "pins": [
        {
          "name": "DFT_STRAP0",
          "mode": "Input",
          "pull_mode": "PullUp"
        },
        {
          "name": "DFT_STRAP1",
          "mode": "Alternate",
          "pull_mode": "PullUp",
        }
      ]
    }
```